tests/e2e/performance/mcg/test_mcg_cosbench.py

```diff
@@ -0,0 +1,85 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import E2ETest, performance
+from ocs_ci.ocs.cosbench import Cosbench
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="function")
+def cosbench(request):
+
+    cosbench = Cosbench()
+
+    def teardown():
+        cosbench.cosbench_teardown()
+
+    request.addfinalizer(teardown)
+    return cosbench
+
+
+@performance
+@pytest.mark.polarion_id("OCS-3694")
+class TestMCGCosbench(E2ETest):
```

Comment:
you should use **PASTest** and not **E2ETest** for performance test,

Reply:
Fixed.